README.md

```diff
@@ -37,25 +37,33 @@ Then, initiate the SmartcarAuth object in the UIViewController.
 ```swift
 let appDelegate = UIApplication.shared.delegate as! AppDelegate
 
-func completionHandler(err: Error?, code: String?, state: String?) -> Any {
+func completionHandler(code: String?, state: String?, err: AuthorizationError?,) -> Void {
  // Receive authorization code
 }
 
 appDelegate.smartcar = SmartcarAuth(
   clientId: "afb0b7d3-807f-4c61-9b04-352e91fe3134",
-  redirectUri: "scafb0b7d3-807f-4c61-9b04-352e91fe3134://page",
+  redirectUri: "scafb0b7d3-807f-4c61-9b04-352e91fe3134://exchange",
   scope: ["read_vin", "read_vehicle_info", "read_odometer"],
   completion: completionHandler
 )
 let smartcar = appDelegate.smartcar
 
-// Launch Connect on the SFSafariViewController
-smartcar.launchAuthFlow(state: state, forcePrompt: false, testMode: false, viewController: viewController)
+// Generate a Connect URL
+let authUrl = authUrlBuilder().build()
+
+// Launch Connect using SFAuthenticationSession (iOS 11) or ASWebAuthenticationSession (iOS 12+)
+smartcar.launchWebAuthSession(url: authUrl, redirectUriScheme: "scafb0b7d3-807f-4c61-9b04-352e91fe3134")
+
+// Launch Connect on the SFSafariViewController if using iOS 10
+smartcar.launchAuthFlow(url: authUrl, viewController: viewController)
 ```
 
 ## Handling the Redirect
 
-The Connect response is returned to the app via the iOS openURL app delegate method, so you need to pipe this through to the current authorization session
+For iOS 11 and above, if you launched Connect using the _launchWebAuthSession_ method, the callback URL with the authentication code (or error) is passed back to the application by the session, and the URL is passed to the completion handler and no further action to intercept the callback is required.
+
+For iOS 10, or if you launched Connect using the _launchAuthFlow_ method, the Connect response is returned to the app via the iOS openURL app delegate method, so you need to pipe this through to the current authorization session.
 
 ```swift
 /**
```

Comment:
Is this code snippet still valid? (also is there a `*` missing on the second line on the doc or is that just the style of docs in swift?)

Reply:
Also remove the `(if any)` in the comment block below

Reply:
yes, I believe it is just the style of swift! I followed the pattern from the example linked from the jazzy docs :+1: